### DISCUSSION TITLE
Rebranding For NexoPOS

### DISCUSSION THREAD

**Discussion by magedfathi** [Blair2004/NexoPOS#1012] [General]
It is a very important to hide the actual software price from the end user or anything will lead to the nexo website because we are not selling installation service to end users but we providing a solution. let us say if customer know the actual price so we cant offer the price we want or easily he can ask anyone to provide installation for another project with low price.
also, the marketing and SEO we are going to spend, it should be for us not for nexopos since we are the one paying for it. as per the new policy you had implemented, we can say that your software is not a free. for example i am reselling Restaurant POS so i should purchase the following 
Gastro Module  45&
NPS Adapter For NexoPOS 4.x  15$
Nexo Print Server 2.6 from codcanyon
Pin Module 20$

Also we should make annual renewal to get the updates which fix the issues which is a must (45+15+20 = 80$)

as per the following the rebranding is a must for us and this will be better also for you because no one will resell if he knows it will be one time selling to customers and any other reseller can get his effort (SE), Campaigns,....) if decided to sell the same product.

Also i would like to advice to make the annual renewal little low price for all modules as a one time payment and it is up to you. 

This is my opinion

**Comment by Blair2004** [comment]
Hi
Thank you for those details. We're planning to enhance the rebranding options. If you have a concrete example of areas that can't be rebranded at the moment, share those.

Regarding the renewal, We'll see how to apply a renewal discount with time.

Kind regards.
